src/components/AstronautsPage/types/astronauts.ts

```diff
@@ -0,0 +1,9 @@
+interface AstronautsAgency {
+  name: string;
+}
+
+export interface Astronauts {
+  name: string;
```

Comment:
use common name for AstronautsAgency  and extend

Reply:
done